swap_meet/vendor.py

```diff
@@ -1,2 +1,82 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = inventory
+        if self.inventory == None:
+            self.inventory = []
+    
+    def add(self, item):
+        if self.inventory == None:
+            self.inventory = []
+            self.inventory.append(item)
+        else:
+            self.inventory.append(item)
+        return item
```

Comment:
Because the last line of the two blocks is the same, you could rewrite the code like this:

```
def add(self, item):
    if self.inventory == None:
        self.inventory = []
    self.inventory.append(item)
    return item
```

Also, your constructor should properly set the inventory as a list, so we probably don't need to check whether it's `None` here.